SMLHelper/Patchers/CraftDataPatcher.cs

```diff
@@ -221,7 +221,7 @@ private static void AddCustomTechDataToOriginalDictionary()
                 if (techDataExists)
                 {
                     techData_Remove.Invoke(techData, new object[] { techType });
-                    Logger.Log($"{techType} TechType already existed in the CraftData.techData dictionary. Original value was replaced.");
+                    Logger.Log($"{techType} TechType already existed in the CraftData.techData dictionary. Original value was replaced.", LogLevel.Debug);
```

Comment:
Let's promote this to a `Warn`.  
Also, this will have to later be reconciled with the changes in #77